packages/react-magic-motion/magic-motion/index.tsx

```diff
@@ -37,10 +38,12 @@ export function MagicMotion({
         layoutDependency,
         transition,
       };
-    },
+    }
   );
 
-  return (
+  return isMotionReduced || disabled ? (
+    <>{children}</>
+  ) : (
     <LayoutGroup>
       <LazyMotion features={domMax}>{motionChildren}</LazyMotion>
     </LayoutGroup>
```

Comment:
This is the conditional render just renders the plain un-animated children when `disabled` or `prefers-reduced-motion:reduce` is set